zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java

```diff
@@ -847,23 +848,27 @@ public Response stopParagraph(@PathParam("noteId") String noteId,
   public Response registerCronJob(@PathParam("noteId") String noteId, String message)
       throws IOException, IllegalArgumentException {
     LOG.info("Register cron job note={} request cron msg={}", noteId, message);
+    ZeppelinConfiguration conf = notebook.getConf();
+    if (conf.isZeppelinNotebookCronEnable()) {
```

Comment:
we should log a warning I think if cron is disabled but someone is trying to register ?